### PR TITLE
ENH: Crosstab support for setting the margins name #15972

### LINKED ISSUE #15972: Crosstab support for setting the margins name
`pandas.pivot_table` already provides the parameter `margins_name ` . `pandas.crosstab` currently does not.

### PR DESCRIPTION
**Pull request by slide333333**
closes #15972

### CODE REVIEW

File: pandas/tools/pivot.py [hunk 1]
```diff
@@ -420,6 +421,9 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
         - If margins is `True`, will also normalize margin values.
 
         .. versionadded:: 0.18.1
+    margins_name : string, default 'All'
```
Comment (TomAugspurger) [inline pandas/tools/pivot.py]: Can you add a ` ..versionadded:: 0.20.0` for this parameter?

**Review by TomAugspurger — CHANGES_REQUESTED** [review]
Thanks.

One comment on the docstring.

Can you also add a note in the whatsnew file, and a couple tests (one with the default, one with a user-specified argument).

**Comment by jreback** [comment]
needs a rebase and tests

**Comment by TomAugspurger** [comment]
Looks like https://github.com/pandas-dev/pandas/pull/16489 fixed this in the meantime. Sorry @slide333333 , but thanks anyways for the PR.

**Pull request closed without merging**